container.te

```diff
@@ -836,6 +836,7 @@ container_spc_stream_connect(container_domain)
 fs_dontaudit_remount_tmpfs(container_domain)
 dev_dontaudit_mounton_sysfs(container_domain)
 dev_dontaudit_mounton_sysfs(container_domain)
+allow container_t tmpfs_t:filesystem { mount };
```

Comment:
Please do 
```
fs_mount_tmpfs(container_domain)
```

Reply:
Thanks for your review!
[Updated](https://github.com/containers/container-selinux/compare/251c530763a9ef789c79985e8536edb73d1f4857..6dff9fd395ffb81298f0d667c2c824ecbf9ebada)